apispec/System_Text_Encoding_UTF8.md

```diff
@@ -0,0 +1,10 @@
+---
+uid: System.Text.Encoding.UTF8
+summary: '*You can override summary for the API here using *MARKDOWN* syntax'
+---
+
+*Please type below more information about this API:*
```

Comment:
I don't think you're supposed to leave this line here. And the same probably goes for `summary` above. (Not sure what is the syntax for not overriding the summary.)